core/domain/stats_jobs_one_off.py

```diff
@@ -889,48 +872,24 @@ def entity_classes_to_map_over(cls):
 
     @classmethod
     def _require_non_negative(
```

Comment:
Please dedupe this method since it's behind used twice. It should work fine as a simple top-level function.

Reply:
Done